packages/@webex/webex-core/src/lib/credentials/scope.js

```diff
@@ -32,3 +34,17 @@ export function filterScope(toFilter, scope) {
     .sort()
     .join(' ');
 }
+
+/**
+ * Returns the scopes not included in the other given scope
```

Comment:
It's not obvious from the function description, parameter decription or variable names, which scope is which, please make it as clear as possible :)

Reply:
Perhaps just change description to 'Returns a string containing all items in scopeA that are not in scopeB, or an empty string if there are none.'

Reply:
I would like to keep the function general, so I picked the later suggestion and updated the docs.